README.md

```diff
@@ -3,6 +3,61 @@
 Introduction To Version Control With Git and GitHub
 ============================
 
+This is a repository to house materials for a 1/2 day workshop introducing participants to version control using git and the popular code hosting platform, GitHub.
+
+The goal of this workshop is to teach new-to-programming data professionals how to harness the power of version control by giving attendees a foundation in the fundamentals of version control and git.
+
+GitHub is a widely-used platform for hosting and sharing code repositories, facilitating seamless collaboration among developers.  Git, on the other hand, is a popular version control system that enables you to track changes in your code, collaborate with team members, and maintain a history of your project's development. 
+
+By the end of this course, you will have gained a fundamental understanding of version control, enabling you to:
+
+- Set up a github account and install git (pre-workshop)
+- Understand the value-add of using version control to manage software projects
+- Understand the basics of how automated version control systems work.
+- Use command-line git to track local file changes
+- (*more about git*)
+- What are remote hosting platforms and why do we use them?
+- Connect to GitHub, create a remote repository and push local changes to GitHub
+- (*Something about BC Gov specific materials Org, Licensing, bcgovr*)
+
+The workshop content will follow best practices for version control such as writing descriptive commit messages, including a .gitignore file and tracking relevent project files.
+
+## Who should take this course?
+
+ * Anyone looking to solidify their foundational understanding of version control
+ * Anyone keen to learn what git or GitHub is (**no prior experience necessary!**)
+ * Anyone wondering how version control can be used to manage and collaborate on software projects
+
+## Previous Offerings
+
+ * November 2023 will be the first offering of this course
```

Comment:
I suggest we remove this section. We may only offer this internal branch lesson 1x, I want to manage expectation (this feels a bit like we are building a course for scaling out).